extension/background/intentRunner.js

```diff
@@ -89,6 +112,68 @@ export class IntentContext {
     });
   }
 
+  async startFollowup(message = undefined) {
+    this.expectsFollowup = true;
+    if (message) {
+      this.acceptFollowupIntent = message.acceptFollowupIntent || [];
+      this.skipSuccessView = message.skipSuccessView || false;
+      this.insistOnFollowup = message.insistOnFollowup || false;
+      this.followupHint = {
+        heading: message.heading,
+        subheading: message.subheading,
+      };
+    }
+    return browser.runtime.sendMessage({
+      type: "handleFollowup",
+      method: "enable",
+      message: {
+        heading: this.followupHint.heading,
+        subheading: this.followupHint.subheading,
+      },
+    });
```

Comment:
Something about this part doesn't seem to be working for me now. The popup doesn't reopen the mic and doesn't show a message, even though it worked previously. 

Actually I'm getting all sorts of peculiar behavior. If I do "search for lions" then it shows the followup, but doesn't actually work. Or, maybe the initial timeout is too low, so when I saw "next" the recorder has already stopped (is there an indicator?)

Then the search gets a little wonky as well, it seems to lose track of the search page in a bunch of cases, and I get missing message handler messages.

Reply:
see https://github.com/mozilla/firefox-voice/pull/1296#issuecomment-607524818, it might fix this. Although not sure what you mean about the missing message handler messages.

Reply:
As for an indicator, we don't have one yet. Currently the timer is set to close the popup after 5 seconds. One issue I ran into with the search intent was how to start the timer *after* the page loaded. I could only get the search results page to wait until load, but saying "next/previous" would not.